README.md

```diff
@@ -1,132 +1,70 @@
-```
-
-# Contract Addresses
+## Setup
 
-The testnet runs on Goerli, while mainnet is on Ethereum Mainnet. The addresses for both of these can be found in `./addresses.json`.
+To set up this project you'll need [git](https://git-scm.com) and [yarn](https://yarnpkg.com/) installed. Note that Yarn v4 is required to install the dependencies and build the project. 
 
-# Local Setup
-
-To setup the contracts locally, checkout the `dev` branch, then run:
+From your command line:
 
 ```bash
-yarn
-yarn build
-```
-
-# Testing
+# Enable Yarn v4
+corepack enable
+yarn set version stable
 
-For testing details see [TESTING.md](./TESTING.md).
+# Clone this repository
+$ git clone https://github.com/graphprotocol/contracts
 
-# Deploying Contracts
+# Go into the repository
+$ cd contracts
 
-In order to run deployments, see [DEPLOYMENT.md](./DEPLOYMENT.md).
+# Install dependencies
+$ yarn
 
-# Interacting with the contracts
-
-There are three ways to interact with the contracts through this repo:
-
-### Hardhat
-
-The most straightforward way to interact with the contracts is through the hardhat console. We have extended the hardhat runtime environment to include all of the contracts. This makes it easy to run the console with autocomplete for all contracts and all functions. It is a quick and easy way to read and write to the contracts.
-
-```
-# A console to interact with testnet contracts
-npx hardhat console --network goerli
+# Build projects
+$ yarn build
 ```
 
-### Hardhat Tasks
-
-There are hardhat tasks under the `/tasks` folder. Most tasks are for complex queries to get back data from the protocol.
-
-### CLI
-
-There is a CLI that can be used to read or write to the contracts. It includes scripts to help with deployment.
-
-## Environment
-
-When running the Hardhat console or tasks you can set what network and accounts to use when sending transactions.
-
-### Network
-
-Selecting a network requires just passing `--network <name>` when running Hardhat. It's important that the network exists in the Hardhat configuration file.
-
-_There is a special network called `localhost` that connects it to a node running on localhost._
-
-### Accounts
-
-The accounts used depends on a few environment variables:
-
-- If MNEMONIC is set you will have available the set of addresses derived from the seed.
-- If PRIVATE_KEY is set, just that account is imported. MNEMONIC always takes precedence over PRIVATE_KEY.
-- If no MNEMONIC or PRIVATE_KEY is set it will use the remote accounts from the provider node.
-- You can always get an account using `ethers.getSigner(<address>)`
-
-Considerations when forking a chain:
+## Documentation
 
-- When running on the `localhost` network it will use by default a deterministic seed for testing purposes. If you want to connect to a local node that is forking while retaining the capability to impersonate accounts or use local accounts you need to set the FORK=true environment variable.
+> Coming soon
```

Comment:
Maybe we could add something like "look at each package's docs in the meantime"?

Reply:
good idea!